storage/inmemory/mutexmap/splits.go

```diff
@@ -233,5 +233,17 @@ func (m *MMSplitStorage) TrafficTypeExists(trafficType string) bool {
 	return exists && value > 0
 }
 
+//Get flag names by sets
+func (m *MMSplitStorage) GetNamesByFlagSets(sets []string) map[string][]string {
+	toReturn := make(map[string][]string)
+	for _, flagSet := range sets {
+		flags := m.flagSets.FlagsFromSet(flagSet)
+		if len(flags) != 0 {
+			toReturn[flagSet] = flags
```

Comment:
sets = [] is valid return, if not it won't be possible logging the warning when set doesn't have splits linked. I would encourage to test that flow as well

Reply:
Updated and added test cases to cover this.